Modules/CaPTkSurvival/cmdapps/CaPTkSurvivalPredictionCmdApp.cpp

```diff
@@ -0,0 +1,201 @@
+/** \file CaPTkSurvivalPredictionCmdApp.cpp
+* \brief CLI program for invoking the Glioblastoma Survival Prediction Index algorithms
+* 
+* These algorithms can be used to train a new survival prediction model
+* or to infer a survival prediction for subjects.
+*/
+
+#include <mitkCommandLineParser.h>
+#include <mitkLogMacros.h>
+
+#include "CaPTkSurvivalPredictionAlgorithm.h"
+
+
+#include <QString>
+#include <QStringList>
+#include <QCoreApplication>
+#include <string>
+#include <map>
+
+
+
+
+
+int main(int argc, char* argv[])
+{
+    QCoreApplication cmdapp(argc, argv); // needed for QCoreApplication::applicationDirPath() to work anywhere
+    // consts placed here for now
+    const QString CBICA_MODEL_DIR = QCoreApplication::applicationDirPath() + QString("/models/survival_model");
```

Comment:
Minor comment...We may not need the survival_model folder...Since these modules/plugins are specific to survival, it should have only survival model(s). Perhaps we could keep the models in the 'models' folder??

Reply:
This is fine as we discussed. Resolving